src/util.ts

```diff
@@ -98,4 +98,12 @@ export namespace Util {
 
     return null
   }
+
+  export function getStartDate() {
+    return new Date(Date.parse(core.getInput('startDate')))
+  }
+
+  export function getEndDate() {
+    return new Date(Date.parse(core.getInput('endDate')))
+  }
```

Comment:
I'm not doing any validation on the input, as `Date.parse` will return `NaN` if the format isn't a date.